tensorflow/core/kernels/reduction_ops_sum.cc

```diff
@@ -46,10 +46,22 @@ REGISTER_KERNEL_BUILDER(
       ReductionOp<GPUDevice, type, Eigen::internal::SumReducer<type>>);
 REGISTER_GPU_KERNELS(float);
 REGISTER_GPU_KERNELS(double);
-REGISTER_GPU_KERNELS(complex64);
-REGISTER_GPU_KERNELS(complex128);
 #undef REGISTER_GPU_KERNELS
 
+REGISTER_KERNEL_BUILDER(
+    Name("Sum")
+        .Device(DEVICE_GPU)
+        .TypeConstraint<complex64>("T")
+        .HostMemory("reduction_indices"),
+    ReductionOp<GPUDevice, complex64, Eigen::internal::SumReducer<complex64>>);
+
+REGISTER_KERNEL_BUILDER(
+    Name("Sum")
+        .Device(DEVICE_GPU)
+        .TypeConstraint<complex128>("T")
+        .HostMemory("reduction_indices"),
+    ReductionOp<GPUDevice, complex64, Eigen::internal::SumReducer<complex64>>);
```

Comment:
Oops, I didn't actually plan to put this into the PR, so I never took a second look at it. Thanks for spotting this.